scripts/run-cargo.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# This is a script for running cargo with a variety of options. It is currently
+# used by the 'x-' makefile targets, for experimenting with compile-time
+# options.
+#
+# The most significant thing this script does is manage cargo config files,
+# moving them into place from `etc/` to `.cargo/`, and cleaning up afterward.
+#
+# It takes its arguments as environment variables just because they are easier
+# than parsing the command line in bash.
+#
+# The "X_" prefixes are to avoid polluting the "CARGO_*" namespace.
+
+# Any error -> exit 1
+set -e
+
+if [[ -e .cargo/config ]]; then
+   rm .cargo/config
```

Comment:
How about change directory first? Otherwise, if I run the script accidentally, like in my home directory, directories may be removed unexpectedly.

Reply:
Good idea. I'll make that change.